{
  "submission_id": "S70",
  "user_id": "U1",
  "problem_id": "P7",
  "timestamp": "2025-09-26T20:24:35Z",
  "status": "runtime_error",
  "language": "python",
  "test_results": [
    {
      "test_case_number": 1,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.04,
      "memory_usage": 7.6484375,
      "actual_output": "",
      "expected_output": "YES",
      "input": "4 12\n2 3 4 9"
    },
    {
      "test_case_number": 2,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.01,
      "memory_usage": 7.87109375,
      "actual_output": "",
      "expected_output": "NO",
      "input": "5 30\n2 5 7 11 13"
    },
    {
      "test_case_number": 3,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.05,
      "memory_usage": 7.9140625,
      "actual_output": "",
      "expected_output": "YES",
      "input": "3 18\n6 9 10"
    }
  ]
}